raiden/network/pathfinding.py

```diff
@@ -0,0 +1,13 @@
+import requests
+from raiden.constants import DEFAULT_HTTP_REQUEST_TIMEOUT
+
+
+def get_pfs_info(url: str) -> object:
```

Comment:
The return type can be annotated better

Reply:
Like `List[PFSInfo]`? (I can choose freely?)

Reply:
done